coordinator/agent/agent.go

```diff
@@ -27,6 +26,15 @@ type LoadAgent struct {
 	client *http.Client
 }
 
+// TODO: maybe move this into a shared model package.
+// agentResponse contains the data returned by load-test agent API.
+type agentResponse struct {
```

Comment:
This was needed to avoid a cyclic inclusion between packages. Didn't want to bring a `model` package so I opted for some minor duplication.